app/src/main/res/layout/fragment_favorites_recipes.xml

```diff
@@ -36,15 +36,6 @@
         android:gravity="center"
         android:text="@string/no_favorites_yet" />
 
-    <ProgressBar
```

Comment:
Why you delete this ProgressBar ?

Reply:
because favorites recipes are recieved from local database and i guess progress bar will never be visible

Reply:
fixed it